src/components/Authentication/LoginPage.js

```diff
@@ -92,75 +122,317 @@ const LoginPage = () => {
         });
     }, [login, email, password]);
 
-    const toggleSavePassword = useCallback(({target}) => {
+    const toggleSavePassword = useCallback(({ target }) => {
         setShouldSave(target.checked);
     }, []);
 
-    return (
-        <Paper className={`${classes.root} ${classes.smallerRoot}`}>
-            <Typography align="center" className={classes.header}
-                color="primary">
-                sign in
-            </Typography>
-            <form onSubmit={handleSubmit}>
-                <TextField error={hasError || email === ""} fullWidth
-                    inputProps={{"data-cy": "emailField"}} label="E-Mail"
-                    margin="normal" onChange={handleTextInput(setEmail)}
-                    value={email} />
-                <PasswordInput autoComplete="current-password"
-                    error={hasError || password === ""} fullWidth
-                    inputProps={{"data-cy": "passwordField"}} label="Password"
-                    onChange={handleTextInput(setPassword)}
-                    value={password} />
-                <Grid alignItems="center" className={classes.options} container
-                    justify="space-between">
-                    <Grid item>
-                        <FormControlLabel
-                            control={<Checkbox checked={shouldSave}
-                                inputProps={{"data-cy": "rememberMe"}}
-                                onChange={toggleSavePassword} />}
-                            label="Remember Me" />
-                    </Grid>
-                    <Grid item>
-                        <Link className={classes.forgot}
-                            data-cy="forgotPassword" to={{
-                                "pathname": "/forgotpassword",
-                                "state": {email},
-                            }}>
-                            Forgot Password?
-                        </Link>
-                    </Grid>
-                </Grid>
-                <Grid alignItems="center" container justify="space-evenly">
-                    <Grid item>
-                        <Button className={classes.primaryButton}
-                            color="primary" data-cy="signInButton"
-                            disabled={!email || !password || loading}
-                            type="submit" variant="contained">
-                            sign in
-                        </Button>
-                    </Grid>
-                    <Grid item>
-                        <Button className={classes.secondaryButton}
+    const handleCheck = () => {
+        setRealEmail(email);
+        getUserType()
+    }
+
+    const renderEmailLogin = () => {
+        return (
+            <div>
```

Comment:
change this to a fragment `<> ... </>` instead of a div. Makes the website heavier

Reply:
I do run a formatter, for some reason mine is bugged and closing tag spacing doesn't work.